releasenotes/notes/dynamic_qpy_with_13_cap-c77411ff7e286c0b.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    The minimal ``qiskit`` version is now 1.3.
+  - |
+    The QPY payload now is generated based on ``qiskit.qpy.QPY_VERSION`` and capped until version 13.
```

Comment:
We switched to using towncrier for our release notes - https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/CONTRIBUTING.md#adding-a-new-release-note. You can add a release note to the `/unreleased` folder with a file name of `2096.upgrade.rst`.

Reply:
oops, fixed in fbdc01efcc6e0006749d29d539c0d7f97cc55dbd